emu/src/cpu/arm7tdmi.rs

```diff
@@ -177,7 +177,7 @@ impl Arm7tdmi {
             LoadStoreImmOffset => self.load_store_immediate_offset(op_code),
             LoadStoreHalfword => self.load_store_halfword(op_code),
             SPRelativeLoadStore => self.sp_relative_load_store(op_code),
-            LoadAddress => unimplemented!(),
+            LoadAddress => self.load_address(op_code),
```

Comment:
Now that you introduced the separation between decode and execution I think that it would be better to write new instructions using that logic

Reply:
I know, is a bit too effort now, I will refactor time by time :)